modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -51,6 +51,8 @@ const ClassicEditor = ({
 			if (CKEDITOR.env.gecko && CKEDITOR.tools.trim(data) === '<br />') {
 				data = '';
 			}
+
+			data = data.replace(/(\u200B){7}/, '');
```

Comment:
I recall seeing patches related to this in the past (not sure if here, or in our CKEditor fork, or in AlloyEditor) with this kind of zero-width space hack, but I can't find any links. It is all a bit like black magic, so I can't claim to really understand it or know whether this change could have unintended side-effects.

So, if tests are green and manual QA looks good (which I trust you've done), I guess we can forward it. Let's give @julien and @carloslancha a chance to respond though, because they might remember something I've forgotten.

IIRC, this is safe to backport because [I don't think IE is going to have any problem](https://caniuse.com/mdn-javascript_grammar_unicode_escape_sequences) with a unicode escape in a regexp literal.

One thing you might do is `(?:\u200b)` instead of `(\u200B)` (ie. use a non-capturing group, seeing as you are doing nothing with the group). Up to you whether you want to do that kind of micro-optimization or not though.

Reply:
I don't seen any problem with this PR either - and thanks @ChrisKian for the fix.

However, I don't recommend using `ClassicEditor` yet, at least not until [this issue](https://github.com/ckeditor/ckeditor4-react/issues/125) is fixed.

Reply:
also adding a link to #390 because it contains useful information about things that don't work correctly with ```ClassicEditor```

Reply:
> However, I don't recommend using ClassicEditor yet, at least not until this issue is fixed.

There's contexts where that doesn't make a difference, so we shouldn't "not recommend" but rather inform of the currently known issues so developers can make judgement decisions on their own ;)

Reply:
@jbalsas Ok understood, I think #390 is a good summary of the known issues.